aspnetcore/tutorials/min-web-api.md

```diff
@@ -0,0 +1,384 @@
+---
+title: "Tutorial: Create a minimal web API with ASP.NET Core"
+author: rick-anderson
+description: Learn how to build a minimal web API with ASP.NET Core.
+ms.author: riande
+ms.date: 8/24/2021
+monikerRange: '>= aspnetcore-6.0'
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR, Models]
+uid: tutorials/min-web-api
+---
+
+# Tutorial: Create a web minimal API with ASP.NET Core
+
+<!-- TODO: Remove aspnetcore\tutorials\min-web-api\samples\6.x -->
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+Minimal APIs are architected to create REST APIs with the minimal dependencies. They are ideal for microservices and apps that want to include only the minimum files, features, and dependencies in ASP.NET Core.
+
+This tutorial teaches the basics of building a miminal web API with ASP.NET Core. For a tutorial on creating a web API project that contains more features, see [Create a web API](xref:tutorials/first-web-api)
```

Comment:
Shouldn't it say minimal API instead of  minimal web api?  That way it is more in line with the blogs.

Reply:
```suggestion
This tutorial teaches the basics of building a miminal web API with ASP.NET Core. For a tutorial on creating a web API project that contains more features, see [Create a web API](xref:tutorials/first-web-api).
```